hack/integration-tests-local.sh

```diff
@@ -9,32 +9,45 @@ export AWS_DEFAULT_REGION=us-east-1
 export AWS_ENDPOINT=localhost:9000
 export AWS_DISABLE_SSL=true
 
-docker container run -d --name helm-s3-minio \
+DOCKER_NAME='helm-s3-minio'
+
+cleanup() {
+  if $(docker container ls | grep -q "${DOCKER_NAME}\$") ; then
+    docker container rm --force --volumes "${DOCKER_NAME}" || :
+  fi
+}
+
+cleanup
+
+on_exit() {
+  if [ -z "${SKIP_CLEANUP}" ]; then
```

Comment:
This test now fails without this variable set: 

```
./hack/integration-tests-local.sh: line 26: SKIP_CLEANUP: unbound variable
```

```suggestion
  if [ -z "${SKIP_CLEANUP:-}" ]; then
```